brick.cabal

```diff
@@ -1,5 +1,5 @@
 name:                brick
-version:             0.36.3
+version:             0.36.4
```

Comment:
Please revert this hunk. I only change the version number at release time, which is the only time it's possible to tell what the appropriate change is (by looking at all the commits in the release).